tracing-distributed/src/trace.rs

```diff
@@ -87,6 +87,21 @@ pub enum TraceCtxError {
     NoParentNodeHasTraceCtx,
 }
 
+impl std::fmt::Display for TraceCtxError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        use TraceCtxError::*;
+        write!(f, "{}",
+            match self {
+                TelemetryLayerNotRegistered => "`TelemetryLayer` is not a registered subscriber of the current Span",
```

Comment:
These messages look OK to me, though I'd defer to @chrisdickinson for organizational opinions here.